docs/getting-started/quickstart.rst

```diff
@@ -404,7 +404,7 @@ We'll first start our server...
     .. code-block:: console
 
         $ javac -cp oso-0.2.5.jar:. Server.java
-        $ java Server
+        $ java -cp oso-0.2.5.jar:. Server
```

Comment:
Someone had a way of replacing text, would be cool to do that here

Reply:
Sorry that's Sam's currently open PR, ignore

Reply:
No, you're right. Just rebased on top of it.